### ISSUE TITLE
Why is this repository deprecated?

### ISSUE THREAD

**Issue by optikos** [servo/cocoa-rs#183]
The README.md should be updated with a sentence or paragraph explaining just a little bit why this repository is deprecated (i.e., what is to replace it) and why abandon this repository’s approach to a projection of Objective C’s framework-API to Rust.

**Comment by jdm** [comment]
You're right. In any case, it's deprecated because the code was merged into https://github.com/servo/core-foundation-rs/tree/master/cocoa which is the new source of truth.

**Comment by optikos** [comment]
Thank you very much.  At least when viewing this issue, others & I now know precisely where to go.
